src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,71 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(toFileName);
+        if (file.exists()) {
+            file.delete();
+        }
+        String[] result = countResult(readFromFile(fromFileName));
+        writeToFile(file,result);
+    }
+
+    public String[] readFromFile(String fileName) {
+        File file = new File(fileName);
+        List<String> arrayList = new ArrayList<>();
+        try {
+            List<String> strings = Files.readAllLines(file.toPath());
+            for (int i = 0; i < strings.size(); i++) {
+                arrayList.add(strings.get(i));
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t read file ", e);
+        }
+        return arrayList.toArray(String[]::new);
+    }
+
+    private String[] countResult(String[] array) {
```

Comment:
the method should return a simple String. 
```suggestion
    private String countResultAndGetReport(String[] array) {
```